File: SADC.Domain/Farm.cs
namespace SADC.Domain
{
    public class Farm
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Location { get; set; }
        public string ImageURL { get; set; }
        public double Size { get; set; }
        public ICollection<EmployeesFarms> Employees { get; set; }
        public ICollection<Field> Fields { get; set; }
    }
}

File: SADC-App/src/app/models/Farm.ts
import { Field } from './Field';
import { Employee } from './Employee';
import { User } from '../models/identity/User';

export interface Farm {
    id: number; 
    name: string; 
    location: string; 
    imageURL: string; 
    size: number; 
    fields: Field[]; 
}
